org.qtodotxt.qtodotxt.json

```diff
@@ -0,0 +1,125 @@
+{
+  "app-id" : "org.qtodotxt.qtodotxt",
+  "runtime" : "org.kde.Platform",
+  "runtime-version" : "5.11",
```

Comment:
5.12 is out since few weeks, would be good to test it.

Reply:
@barthalion 

> 5.12 is out since few weeks, would be good to test it.

I'm not sure if recommending KDE 5.12 is such a good idea.
https://github.com/flathub/flathub/issues/805

Reply:
May be we should continue to use the 5.11 version until a new stable version come.

Reply:
Issues with 5.12 were resolved so it's recommended to switch to it.